accounts-cluster-bench/src/main.rs

```diff
@@ -683,6 +689,8 @@ fn run_accounts_bench(
             let _ = executor.drain_cleared();
         }
 
+        start_bench_tx.send(()).unwrap();
```

Comment:
Sits in the account creation loop, just after the creation block.

Reply:
This will only unblock one worker right? You need a Barrier or you can use a channel and drop the sender to make recv fail (and handle the failure by just continuing)